schema/proto3/port.proto

```diff
@@ -28,13 +28,14 @@ message PortConfiguration {
     
     NetworkType network_type = 2;
     string project_id = 3;
-    string network_id = 4;
-    string id = 5;
-    string name = 6;
-    string network_ns = 7;
-    string mac_address = 8;
-    string veth_name = 9;
-    bool admin_state_up = 10;
+    string subnet_id = 4;
+    uint32 mq_subscription_index = 5;
+    string id = 6;
```

Comment:
Should we move id up a bit?

Reply:
yes, ID is one of the most important field, I will move it up.